modules/c-ares/1.19.1/patches/add_build_file.patch

```diff
@@ -0,0 +1,254 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,251 @@
++load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
++
++config_setting(
++    name = "darwin",
++    values = {"cpu": "darwin"},
```

Comment:
Copied the BUILD file from https://github.com/grpc/grpc/blob/master/third_party/cares/cares.BUILD, but I don't think using cpu value is the recommended way to detect platform anymore?

/cc @pzembrod

Reply:
Moved to use `@platform//os`.